tests/foreman/destructive/test_capsulecontent.py

```diff
@@ -91,3 +92,98 @@ def test_positive_sync_without_deadlock(
 
     sync_status = large_capsule_configured.nailgun_capsule.content_sync(timeout='90m')
     assert sync_status['result'] == 'success', 'Capsule sync task failed.'
+
+
+@pytest.mark.tier4
+@pytest.mark.skip_if_not_set('capsule')
+def test_positive_sync_without_deadlock_after_rpmtrimchangelog(
+    target_sat, large_capsule_configured, function_entitlement_manifest_org
+):
+    """Promote a CV published with larger repos into multiple LCEs, assign LCEs to blank Capsule.
+    Assert that the sync task succeeds and no deadlock happens.
+
+    :id: 91c6eec9-a582-46ea-9898-bdcaebcea2f1
+
+    :setup:
+        1. A blank external capsule that has not been synced yet (!) with immediate download
```

Comment:
It seems you are missing the 'immediate policy' in the test. Honestly, I don't think you would be able to sync both the repos on 100G SAT with immediate. Is it required for this case?

Reply:
SD ACKed no need for immediate policy, so please just update the docstring @Gauravtalreja1 .